Resources/Prototypes/Entities/Objects/Specific/Janitorial/spray.yml

```
@@ -82,10 +82,10 @@
   - type: SolutionContainerManager
     solutions:
       spray:
-        maxVol: 100
+        maxVol: 120 # Carpmosia-edit - extendedReagentStorage
```

This is identical to the parent.
```suggestion
        #        maxVol: 120 # Carpmosia-edit - extendedReagentStorage
```